README.md

```diff
@@ -166,41 +107,26 @@ Argument "INPUT" is required
- -qualityThreshold N                                             : Phred scaled quality threshold used for trimming. If omitted, Phred 20 is used.
  -realign_indels                                                 : Locally realign indels present in reads.
  -recalibrate_base_qualities                                     : Recalibrate the base quality scores (ILLUMINA only)
  -repartition N                                                  : Set the number of partitions to map data to
  -sort_fastq_output                                              : Sets whether to sort the FASTQ output, if saving as FASTQ. False by default.
                                                                    Ignored if not saving as FASTQ.
  -sort_reads                                                     : Sort the reads by referenceId and read position
- -trimBeforeBQSR                                                 : Performs quality based trim before running BQSR. Default is to run quality based
-                                                                   trim after BQSR.
- -trimFromEnd N                                                  : Trim to be applied to end of read.
- -trimFromStart N                                                : Trim to be applied to start of read.
- -trimReadGroup VAL                                              : Read group to be trimmed. If omitted, all reads are trimmed.
- -trimReads                                                      : Apply a fixed trim to the prefix and suffix of all reads/reads in a specific read
-                                                                   group.
-```
-
-The ADAM transform command allows you to mark duplicates, run base quality score recalibration (BQSR) and other pre-processing steps on your data.
+ ```
 
-There are also a number of projects built on ADAM, e.g.
-
-- [RNAdam](https://github.com/bigdatagenomics/RNAdam) provides an RNA pipeline on top of ADAM with isoform quantification and fusion transcription detection
-- [Avocado](https://github.com/bigdatagenomics/avocado) is a variant caller built on top of ADAM for germline and somatic calling
-- [PacMin](https://github.com/bigdatagenomics/PacMin) is an assembler for PacBio reads
-- A `Mutect` port is nearly feature complete
-- Read error correction
-- a graphing and genome visualization library
-- [BDG-Services](https://github.com/bigdatagenomics/bdg-services) is a library for accessing a running Spark cluster through web-services or a [Thrift](https://thrift.apache.org/)- interface
-- [Short read assembly](http://www.github.com/fnothaft/xASSEMBLEx)
-- Variant filtration (train model via `MLlib`)
+The ADAM `transform` command allows you to mark duplicates, run base quality score recalibration (BQSR) and other pre-processing steps on your data.
 
 # Getting Started
 
 ## Installation
 
+### Binary Distributions
+
+Bundled release binaries can be found on our [releases][] page.
```

Comment:
Or wait, I think that would point to https://github.com/bigdatagenomics/adam/releases?